.eslintrc

```diff
@@ -33,6 +33,7 @@
     "react"
   ],
   "rules": {
+    "import/extensions": 0,
```

Comment:
1. This causes merge conflicts with another branch changes.
2. Is this setting directly part of real-time results, if not, why not remove it from the commit, it is a good practice to only include tasks that are directly relevant for your task as part of your commits